performance/src/utils/event_data.cpp

```diff
@@ -229,8 +230,10 @@ void event_data::setup_csv(bool use_acts_geom_source, const detector_type* det,
         m_ptc_to_meas_map[ptc].push_back(meas);
 
         if (!include_silicon_cells) {
-            // Fill measurement to particle map
-            m_meas_to_ptc_map[meas][ptc]++;
+            // Each measurement is created by a single particle unless we use
+            // the clusterization results
+            assert(m_meas_to_ptc_map.find(meas) == m_meas_to_ptc_map.end());
+            m_meas_to_ptc_map[meas][ptc] = 1u;
```

Comment:
Should we only look out for this during debugging? 🤔

To make sure that we insert a new element into the map here, we can use [std::map::insert](https://en.cppreference.com/w/cpp/container/map/insert). It is indeed quite a bit more awkward, but you could do let's say:

```c++
if (m_meas_to_ptc_map.insert({meas, {}}).second == false) {
   throw std::runtime_error(...);
}
if (m_meas_to_ptc_map.at(meas).insert({ptc, 1u}).second == false) {
   throw std::runtime_error(...);
}
```

Not impossible that I would've made a mistake in the example, but something like this could technically work I believe. 🤔

And yeah, the second insert could also be kept as an `operator[]` expression. Since by definition you're filling an empty map there. 🤔